### PR TITLE
#1 - Add calculator 

### PR DESCRIPTION
**Pull request by N1kySSS**
Добавил базовый калькулятор с 4 методами: сложение, вычитание, деление и умножение.

### CODE REVIEW

File: src/main/java/Calculator.java [hunk 1]
```diff
@@ -1,17 +1,23 @@
 public class Calculator {
     public int add(int a, int b){
```
Comment (ImmortalIdiot) [inline src/main/java/Calculator.java:2]: Add the empty line before this line

File: src/main/java/Calculator.java [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
-        //TODO inser your realisation in method add
+        return a + b;
     }
     public int dif(int a, int b){
```
Comment (ImmortalIdiot) [inline src/main/java/Calculator.java:5]: You should separate methods with a blank line between them

File: src/main/java/Calculator.java [hunk 3]
```diff
@@ -1,17 +1,23 @@
...
-        //TODO inser your realisation in method dif
+        return a - b;
     }
     public int div(int a, int b){
```
Comment (ImmortalIdiot) [inline src/main/java/Calculator.java:8]: And here

File: src/main/java/Calculator.java [hunk 4]
```diff
@@ -1,17 +1,23 @@
...
-        //TODO inser your realisation in method div
+        if (b == 0) {
+            return 0;
+        }
+        else {
```
Comment (ImmortalIdiot) [inline src/main/java/Calculator.java:12]: Move else to the previous line

File: src/main/java/Calculator.java [hunk 5]
```diff
@@ -1,17 +1,23 @@
...
+            return a / b;
+        }
     }
     public int times(int a, int b){
-        //TODO inser your realisation in method times
+        return a * b;
     }
-    public int solver(){
-        //TODO inser your realisation in method solver
+    public int solver(int a, int b, int c, int d){
```
Comment (ImmortalIdiot) [inline src/main/java/Calculator.java:19]: Add the empty line before this line

File: src/test/java/CalculatorTest.java [hunk 6]
```diff
@@ -3,29 +3,35 @@
 import static org.junit.jupiter.api.Assertions.*;
 
 class CalculatorTest {
+    Calculator calculator = new Calculator();
```
Comment (ImmortalIdiot) [inline src/test/java/CalculatorTest.java:6]: Add the empty line before this line

File: src/test/java/CalculatorTest.java [hunk 7]
```diff
@@ -3,29 +3,35 @@
...
 
     @Test
     void add() {
-        //TODO inser your realisation in test-method add
+        int result = calculator.add(5,3);
```
Comment (ImmortalIdiot) [inline src/test/java/CalculatorTest.java:10]: Arguments are passed to the method using `,` and space.

Example:
`calculator.add(5, 3)`

File: src/test/java/CalculatorTest.java [hunk 8]
```diff
@@ -3,29 +3,35 @@
...
+        assertEquals(8, result);
```
Comment (ImmortalIdiot) [inline src/test/java/CalculatorTest.java:11]: You can use calculator.add here:
`asserEquals(8, calculator.add(5, 3));`

File: src/test/java/CalculatorTest.java
Comment (ImmortalIdiot) [inline src/test/java/CalculatorTest.java:1]: Add an empty line in the end of the file

File: src/test/java/CalculatorTest.java (hunk 7 above)
Comment (ImmortalIdiot) [reply src/test/java/CalculatorTest.java:10]: Fix it in other places

File: src/test/java/CalculatorTest.java (hunk 8 above)
Comment (ImmortalIdiot) [reply src/test/java/CalculatorTest.java:11]: Fix it in other places

**Review by ImmortalIdiot — CHANGES_REQUESTED** [review]

File: src/test/java/CalculatorTest.java (hunk 8 above)
Comment (N1kySSS) [reply src/test/java/CalculatorTest.java:11]: Это необязательно, так наоборот станет хуже читать код.

File: src/test/java/CalculatorTest.java
Comment (N1kySSS) [reply src/test/java/CalculatorTest.java:1]: done

File: src/test/java/CalculatorTest.java (hunk 7 above)
Comment (N1kySSS) [reply src/test/java/CalculatorTest.java]: done

File: src/test/java/CalculatorTest.java (hunk 6 above)
Comment (N1kySSS) [reply src/test/java/CalculatorTest.java:7]: done

File: src/main/java/Calculator.java (hunk 4 above)
Comment (N1kySSS) [reply src/main/java/Calculator.java]: done

File: src/main/java/Calculator.java (hunk 3 above)
Comment (N1kySSS) [reply src/main/java/Calculator.java]: done

File: src/main/java/Calculator.java (hunk 1 above)
Comment (N1kySSS) [reply src/main/java/Calculator.java]: done

**Review by ImmortalIdiot — APPROVED** [review]
Nice job

**Pull request merged**